reboot_toolkit/toolkit_backend.py

```diff
@@ -287,6 +291,29 @@ def load_games_to_df_from_s3_paths(game_paths: list[str]) -> pd.DataFrame:
             current_game['session_num'] = game_path.split('/')[session_num_idx]
             print(current_game['session_num'].iloc[0])
 
+            if add_ik_joints:
+                if 'time' in current_game.columns:
+                    for coord in ('X', 'Y', 'Z'):
+
+                        current_game[f'neck_{coord}'] = (current_game[f'LSJC_{coord}'] + current_game[f'RSJC_{coord}']) / 2.
+
+                        current_game[f'pelvis_{coord}'] = (current_game[f'LHJC_{coord}'] + current_game[f'RHJC_{coord}']) / 2.
+
+                        current_game[f'torso_{coord}'] = current_game[f'pelvis_{coord}']

```

Comment:
This might be the intended action but just want to point this out for you to verify its not a typo.

Reply:
oh yeah, in the current model the pelvis origin is defined as being coincident with the torso, because we don't have information about pelvis anterior posterior tilt